test/libsolidity/util/ContractABIUtils.cpp

```diff
@@ -313,14 +313,16 @@ dev::solidity::test::ParameterList ContractABIUtils::defaultParameters(size_t co
 	return parameters;
 }
 
-dev::solidity::test::ParameterList ContractABIUtils::failureParameters()
+dev::solidity::test::ParameterList ContractABIUtils::failureParameters(bytes const _bytes)
 {
 	ParameterList parameters;
 
 	parameters.push_back(Parameter{bytes(), "", ABIType{ABIType::HexString, ABIType::AlignNone, 4}, FormatInfo{}});
 	parameters.push_back(Parameter{bytes(), "", ABIType{ABIType::Hex}, FormatInfo{}});
 	parameters.push_back(Parameter{bytes(), "", ABIType{ABIType::UnsignedDec}, FormatInfo{}});
-	parameters.push_back(Parameter{bytes(), "", ABIType{ABIType::String}, FormatInfo{}});
+
+	if (_bytes.size() > 68)
```

Comment:
Could you put the `68` in some const variable with an expressive name?
Maybe it's even possible to calculate this value from its origin parts (function selector, tail ptr and msg length) and assign it to the variable that way?

Reply:
Or maybe just a comment

Reply:
There's a function comment in the header that states this behaviour.

Reply:
Ok, I added another comment here in the implementation. @Marenz